seacatauth/generic.py

```
@@ -125,31 +125,50 @@ def get_bearer_token_value(request):
-def get_access_token_value_from_websocket(request):
+def get_access_token_value_from_websocket(request) -> typing.Optional[typing.Tuple[str, str]]:
+	"""
+	Extract access token from 'Sec-WebSocket-Protocol' header.
+
+	Args:
+		request: aiohttp.web.Request object
+
+	Returns:
+		Token (type, value) tuple, ot None if no token is found in the header.
+	"""
 	token_prefix = "access_token_"
```

Ruff (0.12.2)

Possible hardcoded password assigned to: "token_prefix"

(S105)